app/src/main/java/org/simple/clinic/settings/SettingsModel.kt

```diff
@@ -2,14 +2,16 @@ package org.simple.clinic.settings
 
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
+import org.simple.clinic.appupdate.AppUpdateState
 
 @Parcelize
 data class SettingsModel(
     val applicationId: String,
     val name: String?,
     val phoneNumber: String?,
     val currentLanguage: Language?,
-    val appVersion: String?
+    val appVersion: String?,
+    val appUpdateState: AppUpdateState?
```

Comment:
Since we don't care about the actual content of the `AppUpdateState` and only whether it is `ShowAppUpdate` or not, how about making this a nullable boolean, `isUpdateAvailable` instead?